liftwizard-logging/liftwizard-ansi-color-strip/src/main/java/io/liftwizard/logging/ansi/color/strip/AnsiColorStrip.java

```diff
@@ -59,7 +59,7 @@ public static String strip(String input, Charset charset)
                         false))
         {
             ansiOutputStream.write(bytes);
-            return byteArrayOutputStream.toString();
+            return byteArrayOutputStream.toString(StandardCharsets.UTF_8);
```

Comment:
I fixed this one, instead of StandardCharsets.UTF_8 it should be the parameter charset.